### PR TITLE
Orb tools 11 migration

### PR DESCRIPTION
**Pull request by brivu**
This `PR` migrates the existing orb to `orb tools 11`.

### CODE REVIEW

**Review by Jaryt — APPROVED** [review]

File: .circleci/test-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - orb-tools/pack:
+          filters: *filters
+      - orb-tools/publish:
+          orb-name: circleci/circleci-cli
+          vcs-type: << pipeline.project.type >>
+          pub-type: production
+          requires:
+            - orb-tools/pack
+            - test-executors
+          context: orb-publisher
```
Comment (KyleTryon) [inline .circleci/test-deploy.yml:38]:
```suggestion
          context: orb-publisher
          enable-pr-comment: true
          github-token: GHI_TOKEN
```

File: src/@orb.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 description: >
-  Install and configure the CircleCI command-line interface.
+  This orb enables users to easily install and configure the CircleCI command-line interface.
```
Comment (KyleTryon) [inline src/@orb.yml:4]: Do not use the phrase "This orb", it is too generic, the user already understand this is an orb. Recommend not changing the definition here, the original worked.

**Review by KyleTryon — CHANGES_REQUESTED** [review]

File: src/@orb.yml (hunk 2 above)
Comment (brivu) [reply src/@orb.yml]: While I agree, I had to add the extra words to pass the review tests.

```
(in test file review.bats, line 70)
  `exit 1' failed

Orb description appears short (under 64 characters).
Update the description in ./src/@orb.yml to provide a detailed description of the orb.
Use the orb description to help users find your orb via search. Try describing what use-case this orb solves for.
```

File: .circleci/test-deploy.yml (hunk 1 above)
Comment (brivu) [reply .circleci/test-deploy.yml:38]: Added the extra parameters.

File: src/@orb.yml (hunk 2 above)
Comment (brivu) [reply src/@orb.yml]: I added `Easily` and it met the character requirement 😄 

`Easily install and configure the CircleCI command-line interface.`

**Pull request merged**